### PR TITLE
Copy rubocop custom config from IdP

### PR DESCRIPTION
**Pull request by jgrevich**
__Why__
Rubocop was throwing errors at the Capistrano DSL with my capify branch. We have
already configured this to work correctly on the IdP hence we'll copy that
config.

__How__
Copy the config from the IdP app.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+# https://github.com/bbatsov/rubocop/blob/master/config/disabled.yml
+
 AllCops:
   Include:
     - '**/Gemfile'
     - '**/Rakefile'
   Exclude:
-    - 'bin/**/*'
     - 'db/schema.rb'
-    - 'spec/i18n_spec.rb'
```
Comment (jgrevich) [inline .rubocop.yml]: We should probably keep this right?

File: .rubocop.yml [hunk 2]
```diff
@@ -24,6 +31,8 @@ Metrics/ClassLength:
   Max: 100
   Exclude:
   - spec/**/*
+  - app/controllers/users/confirmations_controller.rb
+  - app/controllers/devise/two_factor_authentication_controller.rb
```
Comment (jgrevich) [inline .rubocop.yml]: 🔫

**Comment by amoose** [comment]
GTG % minor code climate issues

File: .rubocop.yml (hunk 1 above)
Comment (monfresh) [inline .rubocop.yml]: No, the offenses reported for that file are valid. We can fix them in a separate PR.

Comment (monfresh) [inline .rubocop.yml]: Or, you can run `rubocop -a spec/i18n_spec.rb` to auto fix it and check it in here if you want

File: .rubocop.yml (hunk 2 above)
Comment (monfresh) [inline .rubocop.yml]: Yes, please remove these.

**Pull request merged**